cabal.project

```diff
@@ -34,47 +34,26 @@ packages:
          ./plugins/hls-refactor-plugin
          ./plugins/hls-overloaded-record-dot-plugin
 
--- Standard location for temporary packages needed for particular environments
--- For example it is used in the project gitlab mirror to help in the MAcOS M1 build script
--- See https://github.com/haskell/haskell-language-server/blob/master/.gitlab-ci.yml
-optional-packages: vendored/*/*.cabal
+index-state: 2023-10-06T06:12:29Z
 
 tests: True
+test-show-details: direct
 
--- mfsolve has duplicate instances in its test suite
--- See: https://github.com/kuribas/mfsolve/issues/8
-package mfsolve
-  tests: False
```

Comment:
Is this not needed anymore?

Reply:
I can't see how it ever was - unless we had a srp for `mfsolve` at some point.